### PR TITLE
Pan os edit url category fix bug

### PR DESCRIPTION
**Pull request by GuyAfik**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-20146

## Description
Fixed an issue where the **pan-os-edit-custom-url-category** command failed when there were ampersand(s) when *sites* argument was provided or when there were existing sites with ampersand(s) in the site of the existing custom URL category.

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by GuyAfik** [comment]
failed unit-test: https://app.circleci.com/pipelines/github/demisto/content/225394/workflows/a6c435ce-03fb-4976-a22f-9984ee741651/jobs/633231

File: Packs/PAN-OS/ReleaseNotes/1_16_2.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### Palo Alto Networks PAN-OS
+- Fixed an issue where the ***pan-os-edit-custom-url-category*** command failed in some occasions.
```
Comment (JudahSchwartz) [inline Packs/PAN-OS/ReleaseNotes/1_16_2.md:3]: we should be more specific here

**Review by JudahSchwartz — APPROVED** [review]
nice catch!

File: Packs/PAN-OS/ReleaseNotes/1_16_2.md (hunk 1 above)
Comment (GuyAfik) [inline Packs/PAN-OS/ReleaseNotes/1_16_2.md:3]:
```suggestion
- Fixed an issue where the ***pan-os-edit-custom-url-category*** command failed when the URL category had existing sites with the character ampersand.
```

Comment (GuyAfik) [reply Packs/PAN-OS/ReleaseNotes/1_16_2.md]: done, let me know what you think

**Pull request merged**